cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -97,6 +90,18 @@ New Features
   application demonstrating parallel execution of the Spark and MapReduce programs using
   Workflows.
 
+- `CDAP-2923 <https://issues.cask.co/browse/CDAP-2923>`__ -
+  Added support for the WorkflowToken in the Spark programs.
+
+- `CDAP-2936 <https://issues.cask.co/browse/CDAP-2936>`__ -
+  Spark program can now specify resources usage for driver and executor process in distributed mode.
+
+
+**Workflows**
+
+- `CDAP-1983 <https://issues.cask.co/browse/CDAP-1983>`__ -
+  Added example application for processing and analyzing Wikipedia data using Workflows.
```

Comment:
It uses token, yes. But it also features more things as CONDITION and FORK nodes. So I think its better not to put the `WorkflowToken` only in the title.